.travis.yml

```diff
@@ -23,5 +23,11 @@ install:
   - conda install --yes python=$TRAVIS_PYTHON_VERSION pip six protobuf>=3.6.0 absl-py opt_einsum numpy scipy pytest-xdist fastcache
   - pip install jaxlib
   - pip install -v .
+  # The following are needed to test the Colab notebooks and the documentation building
+  - conda install --yes -c conda-forge pandoc ipykernel
+  - conda install --yes sphinx sphinx_rtd_theme nbsphinx jupyter_client matplotlib
 script:
   - pytest -n 1 tests examples -W ignore
+  - if [[ "$TRAVIS_PYTHON_VERSION" > "3" ]]; then
+      sphinx-build -M html docs build;
```

Comment:
It might be a good idea to try setting up separate, dedicated [entry in the build matrix](https://docs.travis-ci.com/user/build-matrix/) for the docs rather than doing it after running the unit tests. This would give faster feedback for doc edits, since you could wait only a few minutes for sphinx to run vs 20+ minutes for the test suite to complete first.

(Certainly we can leave this for a follow-on PR, though!)

Reply:
Will do, filed #1435